cmd/shortener/main.go

```diff
@@ -1,19 +1,41 @@
 package main
 
 import (
+	"database/sql"
+	_ "github.com/jackc/pgx/v5/stdlib"
 	"github.com/poggerr/go_shortener/internal/app/storage"
 	"github.com/poggerr/go_shortener/internal/config"
+	"github.com/poggerr/go_shortener/internal/logger"
 	"github.com/poggerr/go_shortener/internal/routers"
 	"github.com/poggerr/go_shortener/internal/server"
 )
 
 func main() {
 	cfg := config.NewConf()
-	strg := storage.NewStorage(cfg.Path)
+
+	if cfg.DB == "" {
```

Comment:
```suggestion
	if cfg.DB != "" {
	db, err := sql.Open("pgx", cfg.DB)
	server.Server(cfg.Serv, r)
	} else {
```
тут немного нелогичное условие получается